src/index.ts

```diff
@@ -296,15 +352,23 @@ class FileBrowser extends Widget {
       return;
     }
 
-    // Fetch common variables.
-    let items = this._model.items;
-    let nodes = this._nodes;
+    // Check for a breadcrumb hit.
+    if (hitTest(this.node.firstChild as HTMLElement, event.clientX, event.clientY)) {
+      this._handleCrumbEvent(event);
+      return;
+    }
 
-    // Find the target row.
-    let index = hitTestNodes(nodes, event.clientX, event.clientY);
+    // Check for a file item hit.
+    let index = hitTestNodes(this._items, event.clientX, event.clientY);
     if (index === -1) {
       return;
```

Comment:
same. if you are access `firstChild` a lot, you can define a read-only getter which returns the breadcrumbs element.